osm-fudge/__main__.py

```diff
@@ -0,0 +1,16 @@
+import bk_tree
+import sys
+
+file  = open(sys.argv[1], 'r')
+
+initial_word = file.readline()
```

Comment:
What happens if the file is empty?

Reply:
Right now it gives me empty result `['']` But should we exit the program if file is empty?

Reply:
Yeah that is probably a good idea.